DQMOffline/Trigger/python/HiggsMonitoring_cfi.py

```diff
@@ -0,0 +1,103 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.topMonitoring_cfi import topMonitoring
+
+hltHIGmonitoring = topMonitoring.clone()
+hltHIGmonitoring.FolderName = cms.string('HLT/Higgs/default/')
+hltHIGmonitoring.histoPSet.metPSet = cms.PSet(
+  nbins = cms.int32 (  30   ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(  300  ),
+)
+hltHIGmonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.int32 (  60   ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(  300  ),
+)
+hltHIGmonitoring.histoPSet.phiPSet = cms.PSet(
+  nbins = cms.int32 (  32  ),
+  xmin  = cms.double( -3.2 ),
+  xmax  = cms.double(  3.2 ),
+)
+hltHIGmonitoring.histoPSet.etaPSet = cms.PSet(
+  nbins = cms.int32 (  30  ),
+  xmin  = cms.double( -3.0 ),
+  xmax  = cms.double(  3.0 ),
+)
+hltHIGmonitoring.histoPSet.htPSet = cms.PSet(
+  nbins = cms.int32 (   60  ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(  600  ),
+)
+
+# Marina
+hltHIGmonitoring.histoPSet.csvPSet = cms.PSet(
+  nbins = cms.int32 ( 50 ),
+  xmin  = cms.double( 0.0 ),
+  xmax  = cms.double( 1.0  ),
+)
+
+hltHIGmonitoring.applyleptonPVcuts = cms.bool(True)
+hltHIGmonitoring.leptonPVcuts = cms.PSet(
+  dxy = cms.double(   0.5  ),
+  dz  = cms.double(   1.   ),
+)
+
+
+#MET and HT binning
+hltHIGmonitoring.histoPSet.metBinning = cms.vdouble(0,20,40,60,80,100,125,150,175,200)
+hltHIGmonitoring.histoPSet.HTBinning  = cms.vdouble(0,20,40,60,80,100,125,150,175,200,300,400,500,700)
+#Eta binning
+hltHIGmonitoring.histoPSet.eleEtaBinning = cms.vdouble(-2.5,-2.4,-2.3,-2.2,-2.1,-2.0,-1.9,-1.8,-1.7,-1.566,-1.4442,-1.3,-1.2,-1.1,-1.0,-0.9,-0.8,-0.7,-0.6,-0.5,-0.4,-0.3,-0.2,-0.1,0,0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1,1.1,1.2,1.3,1.4442,1.566,1.7,1.8,1.9,2.0,2.1,2.2,2.3,2.4,2.5)
+hltHIGmonitoring.histoPSet.jetEtaBinning = cms.vdouble(-2.4,-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1,2.4)
+hltHIGmonitoring.histoPSet.muEtaBinning  = cms.vdouble(-2.4,-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1,2.4)
+#pt binning
+hltHIGmonitoring.histoPSet.elePtBinning = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+hltHIGmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+hltHIGmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+#Eta binning 2D
+hltHIGmonitoring.histoPSet.eleEtaBinning2D = cms.vdouble(-2.4,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.4)
+hltHIGmonitoring.histoPSet.jetEtaBinning2D = cms.vdouble(-2.4,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.4)
```

Comment:
are you sure about this binning ?
please check w/ JME
in addition, don;t you use jets outside the tracker acceptance ?